src/components/FooterContainer.js

```diff
@@ -11,7 +12,7 @@ const tosUrl = 'http://gatkforums.broadinstitute.org/firecloud/discussion/6819/f
 
 const FooterContainer = ({ children }) => {
   return div({ style: { display: 'flex', flexDirection: 'column', minHeight: '100vh' } }, [
-    div({ style: { flex: 1 } }, [children]),
+    div({ style: { flexGrow: 1, display: 'flex' } }, [children]),
```

Comment:
This is a potentially sweeping change. Did we verify that all pages handle this well? Should this include `flex-direction: column`?

Reply:
Good catch on the direction. I've tested every page.